misc.go

```diff
@@ -112,6 +121,14 @@ func postWithMultipartResponse(ctx context.Context, path, name, fieldname string
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode == 429 {
```

Comment:
Let's use the `http.StatusTooManyRequests` in here https://godoc.org/net/http#pkg-constants

Reply:
@james-lawrence I think @catherinetcai is referring to using naked status code numerics, not returning the standard library's error. I agree using `http.StatusTooManyRequests` is better here.

Reply:
Ha that's what I get for just reading the email.